src/main/java/org/zendesk/client/v2/model/Field.java

```diff
@@ -265,10 +265,10 @@ public String toString() {
         private String name;
         private String value;
 
-        private Option() {
+        public Option() {
```

Comment:
Is there other way to construct Options? You tell me, please. I was developing a project which needed the construction of Options.